website/docs/reference/widgets/list.md

```diff
@@ -41,70 +35,110 @@ You can display static JSON data in the `Items` property for generating the list
 ]
 ```
 
-To display the data in the List widget, add three Text widgets and one Image widget in the first list item.
+Add three Text widgets and one Image widget in the first list item to display the data in the List widget.
 
 Follow the steps below to bind each JSON field to the widgets embedded in the List:
 
-* Select the Image widget and add  `{{currentItem.bookImage}}` under `Image` in the property pane. `currentItem` refers to the data for a particular item. All the list items get populated based on the corresponding data in the JSON object.
+* Select the Image widget and add  `{{currentItem.bookImage}}` under **Image** property. **currentItem** refers to the data for a particular item. All the list items get populated based on the corresponding data in the JSON object.
 * You can now see the image in the list item, as the image widget renders the image available on the URL supplied in JSON.
 
 Similarly, you can bind `bookName`, `bookId`, and `price` to the embedded widgets in the List widget.
```

Comment:
The term embedded widgets here is not clear, can we call it `to the text widgets added in the first list item`

Reply:
updated